scripts/update_contributors.sh

```diff
@@ -11,7 +11,6 @@ projectId="316051"
   echo -e "They helped [develop the app](https://github.com/lwouis/alt-tab-macos/graphs/contributors):\n"
 
   curl https://api.github.com/repos/lwouis/alt-tab-macos/contributors \
-    -H "Authorization: token $GITHUB_TOKEN" |
```

Comment:
It may seem like this is unnecessary, but I actually added this because from the CI machine (TravisCI), requests to github get quota'd and denied. Using a token means it doesn't fail because of that, as it deducted from that token's quota vs from the IP quota.